public/css/product.css

```diff
@@ -11,9 +11,13 @@
   padding: 10px;
   border: 1px solid;
   height: 350px;
-  margin-left: 20px;
 }
 
 h1 {
   margin-left: 80px;
 }
+
+.quantity-selector {
+  display: flex;
+  justify-content: space-between;
```

Comment:
Let's use bootstrap classes for this. I think there's a `d-flex` and `justify-content-between`